src/app/component-library/policy-editor/editor/policy-form-expression-constraint/policy-form-expression-constraint.component.ts

```diff
@@ -7,6 +9,7 @@ import {TreeNode} from '../tree';
 @Component({
   selector: 'policy-form-expression-constraint',
   templateUrl: './policy-form-expression-constraint.component.html',
+  providers: [{provide: DateAdapter, useClass: CustomDateAdapter}],
```

Comment:
please provide this globally in the AppModule or SharedModule where other Angular Material config resides, so it has to be provided only once